### PR TITLE
Pdfreactor Exception message

### PR DESCRIPTION
**Pull request by kave**
- included an additional check for pdfReactor to render a better exception message for local developers 
- fixed imports for PDFreactor in HousingCounselorPDFView breaking the views.

- @kurtw 
@richaagarwal @rosskarchner

### CODE REVIEW

**Comment by rosskarchner** [comment]
hold a sec!

**Comment by rosskarchner** [comment]
👍  made a few tweaks I had been working on in a different branch, but should be good now

File: cfgov/legacy/views.py [hunk 1]
```diff
@@ -13,9 +21,6 @@ def get_render_url(self):
     def get(self, request):
 
         self.request = request
-        if settings.DEBUG and PDFReactor is None:
-            return HttpResponse("PDF Reactor is not configured, can not render %s" % self.get_render_url())
```
Comment (rosskarchner) [inline cfgov/legacy/views.py]: why remove this? (though the capitalization was incorrect, it should be PDFreactor)

Comment (kave) [inline cfgov/legacy/views.py]: yeah the capitalization was causing errors, thought it was related to pdfReactor not loading correctly. i've put it back in,  ~_~ its been a slow morning.

**Comment by kurtw** [comment]
Seems legit

**Pull request merged**